ambariclient/base.py

```diff
@@ -23,6 +23,7 @@
 from ambariclient import events, exceptions, utils
 
 LOG = logging.getLogger(__name__)
+LOG.addHandler(utils.NullHandler())
```

Comment:
I saw you did this elsewhere but I never understood the purpose of it.  Does this just prevent logging to STDOUT by default?